docs/source/getting_started/platform.rst

```diff
@@ -15,188 +15,208 @@ This is exactly why we built the
+            clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+            allowfullscreen>
+        </iframe>
+    </div>
 
-.. figure:: images/dataset_overview.jpg
-    :align: center
-    :alt: Alt text
+.. |br| raw:: html
 
-    Dataset overview on the web platform. Here you see general information
-    such as the number of images or the size of the overall dataset.
+    <br />
 
-.. figure:: images/dataset_home.jpg
-    :align: center
-    :alt: Alt text
-    :figclass: align-center
+|br|
 
-    Dataset overview on the web platform. Here you see generalinformation such 
-    as the number of images or the size of the overall dataset.
+.. note:: 
 
-Upload Samples
-"""""""""""""""
+    Head to our :ref:`tutorials <platform-tutorials-label>` to see the many use-cases of the Lightly Platform.
 
-There are two ways to upload samples to a dataset. You can either use the web 
-front-end of the platform or the Python pip package.
 
-We recommend using the `front-end <https://app.lightly.ai>`_ for simple projects 
-consisting of less than 1'000 samples or 1 GByte in size. Once you create a new 
-dataset you should see an upload field. You can either click on it to open the 
-file browser or drag and drop a list of files or a folder. To upload the 
-selected files you simply click on upload. The upload process typically 
-takes a few minutes.
+Basic Concepts
+-----------------------------------
 
-If you work with larger datasets we recommend 
-you to use the `Python pip package <https://github.com/lightly-ai/lightly>`_.
+The Lightly Platform is built around datasets, tags, embeddings, and image metadata.
 
+Learn more about the different concepts in our `Glossary <https://app.lightly.ai/glossary>`_.
 
-Tag
-^^^^^^^^^^^
 
-If you're familiar with git think of tags as commits. Every tag represents 
-a set of images from your dataset. Initially, a new dataset consists of 
-all images. During the filtering process, you reduce the dataset to the 
-relevant fraction for your work. Tags allow you to keep track of the 
-changes and go back in time. Similar to branches in git you can have 
-multiple child tags from the same tag.
 
-.. figure:: images/webapp_tags.gif
-    :align: center
-    :alt: Alt text
-    :figclass: align-center
-    :scale: 150%
 
-    The marked area on the left side in the web app shows the tag interface. 
-    You can use it to select and create new tags.
+Create a Dataset
+-------------------------
 
-You can interact with your tags by using the left-hand menu of the application. 
-You can only create new tags once you modified your dataset. Once a 
-modification is done you can "freeze" the state by creating a tag by 
-giving it a name (text field at bottom) and pressing enter.
 
-There is a linear bar indicating the current number of 
-active samples you work with. 
+To create a new dataset of images with embeddings use the lightly :ref:`lightly-command-line-tool`:
```

Comment:
maybe mention to just add it in the platform, as this are the docs for the platform, no?

Reply:
can you phrase it out? I'm not sure what you mean